### PR TITLE
BUGFIX: BB-1091 BB-1144 Fix of localization issues

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/647/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/226/ : SUCCESS in 1m 44s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/126/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/227/ : SUCCESS in 1m 51s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/111/ : ABORTED in 3m 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/16/ : ABORTED in 4m 00s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/657/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/234/ : SUCCESS in 3m 35s

**Comment by petrjanu** [comment]
Closing as "Gesamt" is wrong translation. Original "Summe" is correct. See https://jira.intgdc.com/browse/BB-943 or https://jira.intgdc.com/browse/BB-544 for further info.

CC @sitina

File: src/translations/de-DE.json [hunk 1]
```diff
@@ -6,17 +6,17 @@
    "visualizations.desc": "Absteigend",
    "visualizations.of": "{page} von {pagesCount}",
    "visualizations.totals.dropdown.heading": "Neue Zeile hinzufügen",
-   "visualizations.totals.dropdown.title.sum": "Gesamt",
```
Comment (petrjanu) [inline src/translations/de-DE.json]: This should be "Summe"

File: src/translations/de-DE.json [hunk 2]
```diff
@@ -6,17 +6,17 @@
...
+   "visualizations.totals.dropdown.title.sum": "Summe",
    "visualizations.totals.dropdown.title.avg": "Mittelwert",
    "visualizations.totals.dropdown.title.min": "Min",
    "visualizations.totals.dropdown.title.max": "Max",
    "visualizations.totals.dropdown.title.nat": "Rollup (Gesamt)",
    "visualizations.totals.dropdown.title.med": "Medianwert",
-   "visualizations.totals.row.title.sum": "Gesamt",
```
Comment (petrjanu) [inline src/translations/de-DE.json]: This should be "Summe"

File: src/translations/de-DE.json [hunk 3]
```diff
@@ -6,17 +6,17 @@
    "visualizations.desc": "Absteigend",
    "visualizations.of": "{page} von {pagesCount}",
    "visualizations.totals.dropdown.heading": "Neue Zeile hinzufügen",
-   "visualizations.totals.dropdown.title.sum": "Gesamt",
+   "visualizations.totals.dropdown.title.sum": "Summe",
    "visualizations.totals.dropdown.title.avg": "Mittelwert",
    "visualizations.totals.dropdown.title.min": "Min",
    "visualizations.totals.dropdown.title.max": "Max",
    "visualizations.totals.dropdown.title.nat": "Rollup (Gesamt)",
```
Comment (petrjanu) [inline src/translations/de-DE.json]: This is correct "Rollup (Gesamt)"

File: src/translations/de-DE.json [hunk 4]
```diff
@@ -6,17 +6,17 @@
...
    "visualizations.totals.dropdown.title.med": "Medianwert",
-   "visualizations.totals.row.title.sum": "Gesamt",
+   "visualizations.totals.row.title.sum": "Summe",
    "visualizations.totals.row.title.avg": "Mittelwert",
    "visualizations.totals.row.title.min": "Min",
    "visualizations.totals.row.title.max": "Max",
-   "visualizations.totals.row.title.nat": "Gesamt",
```
Comment (petrjanu) [inline src/translations/de-DE.json]: This is correct "Gesamt"

**Pull request closed without merging**